src/functionalTest/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterTest.java

```diff
@@ -77,6 +88,16 @@ public void should_return_200_when_single_letter_is_sent() throws Exception {
         verify(insights).trackMessageAcknowledgement(any(Duration.class), eq(true), anyString());
     }
 
+    @Test
+    public void should_return_200_when_same_letter_is_sent_twice() throws Exception {
+        given(queueClientSupplier.get()).willReturn(queueClient);
+
+        String letter = readResource("letter.json");
+
+        send(letter).andExpect(status().isOk());
+        send(letter).andExpect(status().isOk());
```

Comment:
what's the UUID?

Reply:
the assertion is left out in the pending integration test code in another branch